### PR TITLE
Improve the quick start guide to make the start order more clear.

### PR DESCRIPTION
**Pull request by fmauch**
This came up in #112

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,10 +178,13 @@ an error during startup, but will remain usable.
 
 For more information on the launch file's parameters see its own documentation.
 
-Once the robot driver is started, load the previously generated program on the robot panel and
-execute it. From that moment on the robot is fully functional. You can make use of the pause
-function or even stop the program. Simply press the play button again and the ROS driver will
-reconnect.
+Once the robot driver is started, load the previously generated program on the robot panel that will
```
Comment (gavanderhoorn) [inline README.md]: Which program are we referring to here?

File: README.md [hunk 2]
```diff
@@ -178,10 +178,13 @@ an error during startup, but will remain usable.
...
+start the `External Control` programnode and execute it. From that moment on the robot is fully
```
Comment (gavanderhoorn) [inline README.md]:
```suggestion
start the *External Control* program node and execute it. From that moment on the robot is fully
```

File: README.md [hunk 3]
```diff
@@ -178,10 +178,13 @@ an error during startup, but will remain usable.
...
+functional. You can make use of the pause function or even stop the program. Simply press the play
```
Comment (gavanderhoorn) [inline README.md]:
```suggestion
functional. You can make use of the *Pause* function (▶) or even *Stop* the program (◾). Simply press the *Play*
```

File: README.md [hunk 4]
```diff
@@ -178,10 +178,13 @@ an error during startup, but will remain usable.
...
+button again and the ROS driver will reconnect.
```
Comment (gavanderhoorn) [inline README.md]:
```suggestion
button (▶) again and the ROS driver will reconnect.
```

File: README.md [hunk 5]
```diff
@@ -178,10 +178,13 @@ an error during startup, but will remain usable.
...
+
+Inside the ROS terminal running the driver you should see the output `Robot ready to receive control commands.`
```
Comment (gavanderhoorn) [inline README.md]:
```suggestion
Inside the ROS terminal running the driver you should see the output *Robot ready to receive control commands*.
```

Comment (fmauch) [reply README.md:186]: I decided against that, as we are referring to actual terminal output here.

File: README.md [hunk 6]
```diff
@@ -158,7 +158,7 @@ We recommend keeping calibrations for all robots in your organization in a commo
 [package's documentation](ur_calibration/README.md) for details.
 
 #### Quick start
-Once the driver is built and the **externalcontrol** URCap is installed and running on the robot, you are good
+Once the driver is built and the **externalcontrol** URCap is installed on the robot, you are good
 to go ahead starting the driver. (**Note**: We do recommend, though, to calibrate your robot first.)
```
Comment (gavanderhoorn) [inline README.md:162]: Nit, but: we're not actually calibrating the robot, are we? It's more exporting the already present calibration data and converting it to be usable with ROS tools.

Comment (fmauch) [reply README.md:162]: true...

Comment (fmauch) [reply README.md]: Fixed in 8c86b7e

**Pull request merged**